detections/network/windows_gather_victim_network_info_through_ip_check_web_services.yml

```
@@ -18,7 +18,10 @@ search: '`sysmon` EventCode=22  QueryName IN ("*wtfismyip.com", "*checkip.*", "*
   "*ipinfo.io", "*api.ipify.org", "*icanhazip.com", "*ip.anysrc.com","*api.ip.sb",
   "ident.me", "www.myexternalip.com", "*zen.spamhaus.org", "*cbl.abuseat.org", "*b.barracudacentral.org",
   "*dnsbl-1.uceprotect.net", "*spam.dnsbl.sorbs.net", "*iplogger.org*", "*ip-api.com*",
-  "*geoip.*", "*icanhazip.*") |  stats  min(_time) as firstTime max(_time) as lastTime
+  "*geoip.*", "*icanhazip.*", "*ipwho.is*", "*ifconfig.me*", "*myip.com*", "*ipstack.com*", "*myexternalip.com*", 
+  "*ip-api.io*", "*trackip.net*", "*ipgeolocation.io*", "*ipfind.io*", "*freegeoip.app*", "*ipv4bot.whatismyipaddress.com*", 
+  "*hacker-target.com/iptools*") 
+  |  stats  min(_time) as firstTime max(_time) as lastTime
```

Removing the `hacker-target.com/iptools` string as it a URI, and those are not captured by Sysmon EID 22
```suggestion
  "*geoip.*", "*icanhazip.*", "*ipwho.is*", "*ifconfig.me*", "*myip.com*", "*ipstack.com*", "*myexternalip.com*", 
  "*ip-api.io*", "*trackip.net*", "*ipgeolocation.io*", "*ipfind.io*", "*freegeoip.app*", "*ipv4bot.whatismyipaddress.com*") 
  |  stats  min(_time) as firstTime max(_time) as lastTime
```